amass/utils/misc_test.go

```diff
@@ -0,0 +1,99 @@
+package utils
+
+import (
+	"fmt"
```

Comment:
Is anything from `fmt` being used?

Reply:
Left over from debugging, deleted the import.